index.html

```diff
@@ -1088,6 +1089,257 @@ <h4>Nonce</h4>
+
+      <p>
+The BBS+ Bound <a>signature suite</a> 2020 adds a mechanism for recipient
+binding, and is otherwise identical, to the BBS+ <a>signature suite</a>
+2020. It MUST be used in conjunction with the signing and verification
+algorithms in the Linked Data Proofs [[LD-PROOFS]] specification. The suite
+consists of the following algorithms:
+      </p>
+      <table class="simple">
+        <thead>
+        <th>Parameter</th>
+        <th>Value</th>
+        <th>Specification</th>
+        </thead>
+        <tbody>
+        <tr>
+          <td><a>canonicalization algorithm</a></td>
+          <td>https://w3id.org/security#URDNA2015</td>
+          <td>[[RDF-DATASET-NORMALIZATION]]</td>
+        </tr>
+        <tr>
+          <td><a>statement digest algorithm</a></td>
+          <td>Blake2b</td>
+          <td>[[BLAKE2]]</td>
+        </tr>
+        <tr>
+          <td><a>signature algorithm</a></td>
+          <td>BBS+ Signature</td>
+          <td>[[BBS]]</td>
+        </tr>
+        <tr>
+          <td><a>curve name</a></td>
+          <td>BLS12-381</td>
+          <td>[[PAIRING-FRIENDLY-CURVES]]</td>
+        </tr>
+        </tbody>
+      </table>
+
+      <section>
+        <h3>Modification to Algorithms</h3>
+
+        <section>
+          <h4>Create Verify Data Algorithm</h4>
+
+          <p>
+The create verify data algorithm defined below is identical to the create verify
+data algorithm defined for the BBS+ <a>signature suite</a> 2020, with the
+addition of steps to include recipient binding.
+          </p>
+
+          <p>
+The algorithm defined below outlines the process of obtaining the data in the
+form required for both signing and verifying, plus the data required to bind to
+a recipient
+          </p>
+          <ol class="algorithm">
+            <li>
+Obtain a blind message which is a commitment to a secret value as described in
+[[BBS]] section 3.9. This is the recipient binding commitment.
+            </li>
+            <li>
+Initialize an empty array of length equal to the number of statements not
+including the recipient binding commitment; let this be known as the
+<a>statement digests</a> array.
+            </li>
+            <li>
+Use the <a>canonicalization algorithm</a> to canonicalize the input document to a
+set of <a>statements</a> represented as <a>n-quads</a>.
+            </li>
+            <li>
+For each <a>statement</a> in order:
+              <ol class="algorithm">
+                <li>
+Apply the <a>statement digest algorithm</a> to obtain a <a>statement digest</a>
+                </li>
+                <li>
+Insert the <a>statement digest</a> into the <a>statement digests</a> array which
+MUST maintain same order as the order of the statements returned from the
+<a>canonicalization algorithm</a>.
+                </li>
+              </ol>
+            </li>
+            <li>
+Return the <a>statement digests</a> array and the recipient binding commitment.
```

Comment:
Maybe the combination of the statement digests array and the recipient binding commitment should be known as something e.g sign(ing)/verify(ing) data?